src/quo2/components/navigation/bottom_nav_tab/view.cljs

```diff
@@ -1,10 +1,11 @@
-(ns quo2.components.navigation.bottom-nav-tab
+(ns quo2.components.navigation.bottom-nav-tab.view
```

Comment:
Moved content from previous namespace to new files split in `.view` and `.styles`